core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-reports/dot-experiments-reports.component.html

```diff
@@ -132,7 +132,7 @@
                                     [ngClass]="{ 'prospect-winner': row.isWinner }"
                                 >
                                     <button
-                                        class="p-button p-button-sm p-button-text"
+                                        class="p-button-sm p-button-secondary no-padding"
```

Comment:
Does the `no-padding` looks good in the hover? there is no button with no padding in the design system.

Reply:
not needed removed.